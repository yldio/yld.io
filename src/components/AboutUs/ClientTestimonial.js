import React from 'react'
import { Padding } from 'styled-components-spacing'

import { Row, Col, Grid } from '../grid'
import { SectionTitle } from '../Typography'
import GreyBackground from '../Common/GreyBackground'
import VideoSection from '../Common/VideoSection'

const ClientTestimonial = ({ title, video }) => (
  <GreyBackground>
    <Grid>
      <Padding
        top={{ smallPhone: 3, tablet: 4 }}
        bottom={{ smallPhone: 3.5, tablet: 5 }}
      >
        <Padding bottom={{ smallPhone: 3, tablet: 4 }}>
          <Row>
            <Col width={[1, 1, 1, 1, 6 / 12, 6 / 12, 5 / 12]}>
              <SectionTitle>{title}</SectionTitle>
            </Col>
          </Row>
        </Padding>
        <VideoSection src={video.link} />
      </Padding>
    </Grid>
  </GreyBackground>
)

export default ClientTestimonial
