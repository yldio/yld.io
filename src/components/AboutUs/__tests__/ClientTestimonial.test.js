import React from 'react'
import { render } from '../../../utils/tests/test-utils'
import ClientTestimonial from '../ClientTestimonial'

describe('<ClientTestimonial />', () => {
  it('renders correctly', () => {
    const title = 'Testimonial Video Title'
    const video = {
      link: 'https://www.youtube.com/watch?v=Jv5J6a7QJug'
    }
    const { container } = render(
      <ClientTestimonial title={title} video={video} />
    )

    expect(container.firstChild).toMatchSnapshot()
  })
})
