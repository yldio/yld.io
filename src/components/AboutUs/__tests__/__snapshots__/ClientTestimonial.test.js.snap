// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ClientTestimonial /> renders correctly 1`] = `
<section
  class="sc-gZMcBi fwbGBT"
>
  <div
    class="sc-htpNat toUCL"
  >
    <div
      class="sc-bwzfXH egEeOi"
    >
      <div
        class="sc-bwzfXH bXntSd"
      >
        <div
          class="sc-ifAKCX dJZmHe"
        >
          <div
            class="sc-bxivhb ljCSNa"
            width="1,1,1,1,0.5,0.5,0.4166666666666667"
          >
            <h2
              class="sc-EHOje hObgUw"
            >
              Testimonial Video Title
            </h2>
          </div>
        </div>
      </div>
      <div
        class="sc-fjdhpX gNHsWM"
      >
        <div
          class="sc-bxivhb bXsrfg"
          width="1,1,1,0.8333333333333334"
        >
          <div
            class="sc-VigVT jFeVeN"
            height="480"
            width="854"
          >
            <iframe
              allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture"
              allowfullscreen=""
              class="sc-jTzLTM kxEMrN"
              frameborder="0"
              src="https://www.youtube.com/watch?v=Jv5J6a7QJug"
            />
          </div>
        </div>
      </div>
    </div>
  </div>
</section>
`;
